CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Support for OTLP protocol configuration via environment variables:
+  - `OTEL_EXPORTER_OTLP_PROTOCOL` to specify the protocol used by OTLP exporters.
```

Comment:
The changelog entry does not say that the new functionality works only for the OTLP trace exporters.